src/craig_bampton.jl

```diff
@@ -20,19 +20,22 @@ function craig_bampton(K, M, r, l, n)
     X1[abs.(X1) .< eps_] = 0
     X = X1[:,1:n]
     V = X1'*Kll*X1
+    V[abs.(V) .< eps_] = 0
     Kmm = X'*Kll*X
     Kmm[abs.(Kmm) .< eps_] = 0
-    Kbb = Krr + Krl*(-X1*inv(V)*X1'*Klr)
+    b = -X*inv(Kmm)*X'*Klr
```

Comment:
I am sure that we need these manipulations, since Julia does not automatically understand very small values (1.0E-10 for instance) as zeros. This becomes a problem when calculating inverses etc. when the matrix is to only have values on its diagonal.